### PR TITLE
[ESS][8.13] Indicator match rule alert suppression docs 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/security-docs/issues/4715

Previews:
- [Suppress detection alerts](https://security-docs_bk_4888.docs-preview.app.elstc.co/guide/en/security/master/alert-suppression.html)
- [Create rule API | Optional alert suppression fields for query, threshold, and indicator match rules](https://security-docs_bk_4888.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#opt-fields-alert-suppression-create)
- [Update rule API | Optional alert suppression fields for query, threshold, and indicator match rules](https://security-docs_bk_4888.docs-preview.app.elstc.co/guide/en/security/master/rules-api-update.html#opt-fields-alert-suppression-update)

Twin serverless PR: https://github.com/elastic/staging-serverless-security-docs/pull/297

### CODE REVIEW

File: docs/detections/alert-suppression.asciidoc [hunk 1]
```diff
@@ -81,3 +92,10 @@ With alert suppression, detection alerts aren't created for the grouped source e
 image::images/timeline-button.png[Investigate in timeline button, 200]
 
 * Alert details flyout — Select *Take action* -> *Investigate in timeline*.
+
+=== Alert suppression limit by rule type
+
+Except for custom query rules (which don't have an alert suppression limit), each rule type has a maximum number of alerts that can be suppressed:
+
+* **Threshold** - The maximum number of alerts that can be suppressed is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100`.
```
Comment (nastasha-solomon) [inline docs/detections/alert-suppression.asciidoc:100]:
```suggestion
* **Threshold** - The maximum number of is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100` by default.
```

File: docs/detections/alert-suppression.asciidoc [hunk 2]
```diff
@@ -81,3 +92,10 @@ With alert suppression, detection alerts aren't created for the grouped source e
...
+* **Indicator match** - The maximum number is five times the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `500`.
```
Comment (nastasha-solomon) [inline docs/detections/alert-suppression.asciidoc:101]:
```suggestion
* **Indicator match** - The maximum number is five times the value specified for the <<opt-fields-all,`max_signals`>> setting. The default `max_signals` value is `100`, which means the default maximum limit for indicator match rules is `500`.
```

File: docs/detections/alert-suppression.asciidoc [hunk 3]
```diff
@@ -81,3 +92,10 @@ With alert suppression, detection alerts aren't created for the grouped source e
 image::images/timeline-button.png[Investigate in timeline button, 200]
 
 * Alert details flyout — Select *Take action* -> *Investigate in timeline*.
+
+=== Alert suppression limit by rule type
+
+Except for custom query rules (which don't have an alert suppression limit), each rule type has a maximum number of alerts that can be suppressed:
```
Comment (joepeeples) [inline docs/detections/alert-suppression.asciidoc:98]: Maybe reorder; the focus point (each rule type has a maximum) feels a little buried behind two different qualifying phrases.
```suggestion
Some rule types have a maximum number of alerts that can be suppressed (custom query rules don't have a suppression limit):
```

**Review by joepeeples — COMMENTED** [review]
One small edit suggestion. Looks good!

File: docs/detections/api/rules/rules-api-create.asciidoc [hunk 4]
```diff
@@ -523,6 +523,26 @@ preview::[]
 
 |==============================================
 
+====== Indicator match rule
+
+[width="100%",options="header"]
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/api/rules/rules-api-create.asciidoc:528]: I considered this as well, but wasn't sure if it'd be easier to miss when users were searching through the page or our docset. That said, this is the first sub-section within the section so users wouldn't need to really search/scroll at all once they arrived at this section. Plus, combining duplicate information is always for maintenance, so I don't see any major downsides to updating the section header to reflect that these params apply for both custom query rules and indicator match rules.

[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/api/rules/rules-api-create.asciidoc]: Updated in [f96c975](https://github.com/elastic/security-docs/pull/4888/commits/f96c9755f703677278e23353fd70d05b0a3eb96d)

File: docs/detections/alert-suppression.asciidoc [hunk 5]
```diff
@@ -22,35 +23,45 @@ NOTE: Alert suppression is not available for Elastic prebuilt rules. However, if
...
+* Custom query rule: In *Suppress alerts by*, enter 1-3 field names to group events by the fields' values. 
+* Threshold rule: In *Group by*, enter up to 3 field names to group events by the fields' values, or leave the setting empty to group all qualifying events together. 
 --
 +
 NOTE: If you specify a field with multiple values, an alert grouping is created for each value. For example, if you suppress alerts by `destination.ip` of `[127.0.0.1, 127.0.0.2, 127.0.0.3]`, alerts will be suppressed separately for each value of `127.0.0.1`, `127.0.0.2`, and `127.0.0.3`. 
 
+* Indicator match rule: In *Suppress alerts by*, enter 1-3 field names to group events by the fields' values. 
++
+NOTE: If you specify a field with multiple values, an alert grouping is created for alerts that contain the array you specified. For example, if you suppress alerts by `destination.ip` of `[127.0.0.1, 127.0.0.2, 127.0.0.3]`, alerts that contain this array are grouped and suppressed.
+
```
Comment (vitaliidm) [inline docs/detections/alert-suppression.asciidoc:40]: Wondering whether we can improve this section a bit

Screenshot 2024-03-11 at 10 17 29

First not is applicable for both query and threshold rule types.

Second note - only to IM rule type

There is an Intendation  for a second note, but would it be understood by users the first not applied to query rule as well?

Comment (nastasha-solomon) [reply docs/detections/alert-suppression.asciidoc:44]: Combined the two notes and updated the format in [943cc12](https://github.com/elastic/security-docs/pull/4888/commits/943cc12315b52e4523e741e1182a476ad84a288c). 

@vitaliidm let me know if the changes work - thanks!

**Review by vitaliidm — DISMISSED** [review]

File: docs/detections/alert-suppression.asciidoc [hunk 6]
```diff
@@ -81,3 +96,10 @@ With alert suppression, detection alerts aren't created for the grouped source e
 image::images/timeline-button.png[Investigate in timeline button, 200]
 
 * Alert details flyout — Select *Take action* -> *Investigate in timeline*.
+
+=== Alert suppression limit by rule type
+
+Some rule types have a maximum number of alerts that can be suppressed (custom query rules don't have a suppression limit):
+
+* **Threshold** - The maximum number of is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100` by default.
```
Comment (benironside) [inline docs/detections/alert-suppression.asciidoc:104]:
```suggestion
* **Threshold** - The maximum number of alerts is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100` by default.
```

**Review by benironside — APPROVED** [review]
Found one minor bug otherwise LGTM

**Review by joepeeples — APPROVED** [review]

**Pull request merged**